resources/js/components/GraphqlMutation.vue

```diff
@@ -128,6 +131,11 @@ export default {
                 )
 
                 if (response.data.errors) {
+                    if (this.errorCallback) {
+                        await this.errorCallback(this.data, response)
+                        return
```

Comment:
Do we want to block the logout redirect and notifications when we have an error callback added?

Do note that it might also be possible to watch `this.error` depending on your goal